tests/api/test_files.py

```diff
@@ -142,7 +143,9 @@ def test_download(api_client, uploaded_file_ids, filename):
         os.path.dirname(os.path.abspath(__file__)),
         "..",
         "data",
-        original_filename.encode("utf-8"),
+        original_filename.encode("utf-8")
```

Comment:
* on Python 2 the programmer is responsible for encoding non ASCII file names into the file system encoding.
* on Python 3 this is taken care of underlying Python stdlib, only `str` instances are accepted, not `bytes`.